docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -2,227 +2,227 @@
-blockchain to this URL.
+     The ``xo`` client sends requests to update and query the blockchain to the
+     URL of the REST API (by default, ``http://127.0.0.1:8008``).
 
+     If the REST API's URL is not ``http://127.0.0.1:8008``, you must add the
+     ``--url`` argument to each ``xo`` command in this procedure.
 
-.. seealso::
-  :doc:`/cli/xo`
+     The following example specifies the URL for the Docker demo application
+     environment when creating a new game:
 
-Playing XO with the XO Client (Example)
-=======================================
+     .. code-block:: console
 
-Now that we have gone through the basics of the *XO* transaction processor, we are ready
-to play a game. The steps below show you how to set up and play a game using the *XO* CLI.
+        $ xo create my-game --username jack --url http://rest-api:8008
 
-Start the Necessary Components
-------------------------------
 
-To play *XO*, ensure that the following components are running and connected:
-
-#. At least one validator
-#. An *XO* family transaction processor
-#. The REST API
-
-Create Players
---------------
+Step 1. Create Players
+----------------------
 
 Create keys for two players to play the game:
 
 .. code-block:: console
 
     $ sawtooth keygen jack
-    $ sawtooth keygen jill
-
-
-The command produces output similar to the following for both players:
-
-.. code-block:: console
-
     writing file: /home/ubuntu/.sawtooth/keys/jack.priv
     writing file: /home/ubuntu/.sawtooth/keys/jack.addr
+
+    $ sawtooth keygen jill
     writing file: /home/ubuntu/.sawtooth/keys/jill.priv
     writing file: /home/ubuntu/.sawtooth/keys/jill.addr
 
 
-Create a Game
--------------
+.. note::
 
-Create a game with the following command:
+   The output may differ slightly from this example.
 
-.. code-block:: console
 
-    $ xo create game --username jack
+Step 2. Create a Game
+---------------------
 
-To see list of the created games, enter the following command:
+Create a game named ``my-game`` with the following command:
 
 .. code-block:: console
 
-    $ xo list
+    $ xo create my-game --username jack
+
+.. note::
 
-The command outputs a list of the games that have been created:
+   The ``--username`` argument is required for ``xo create`` and ``xo take``
+   so that a single user (you) can play as two players. By default,
+   ``<username>`` is the player's Linux user name.
+
+Verify that the ``create`` transaction was committed by displaying the list of
+existing games:
 
 .. code-block:: console
 
+    $ xo list
     GAME            PLAYER 1        PLAYER 2        BOARD     STATE
-    game                                            --------- P1-NEXT
+    my-game                                         --------- P1-NEXT
 
+.. note::
 
-Take a Space as Player One
---------------------------
+   The ``xo list`` command provides a quick way to show game state
+   instead of requesting the state from the REST API.
```

Comment:
I think that's what `xo list` does. It calls the rest api to retrieve state under the xo namespace. So that saves the user from curling the URL of the rest API. I just don't want to confuse anyone that this list command has local storage or is doing anything other than convenience wrapping the rest API call.

Reply:
💡 ( = I'll change the wording to clarify this.  Thanks!)